src/main/kotlin/com/workos/usermanagement/builders/AuthenticationWithCodeOptionsBuilder.kt

```diff
@@ -0,0 +1,44 @@
+package com.workos.usermanagement.builders
+
+import com.workos.usermanagement.types.AuthenticationAdditionalOptions
+import com.workos.usermanagement.types.AuthenticationWithCodeOptions
+
+/**
+ * Builder for options when creating Authorization URLs with code.
```

Comment:
AFAICT, this constructs the options to be sent in the body of the request rather than a URL. Maybe this?

```suggestion
 * Builder for options when authentication with an authorization code.
```

Reply:
👍